### PR TITLE
Misc cleanup in scripts/

### PR DESCRIPTION
**Pull request by saschanaz**
#### Summary

Random things in scripts/

### CODE REVIEW

File: scripts/migrations/002-remove-webview-flags.test.js [hunk 1]
```diff
@@ -3,10 +3,7 @@
  * http://creativecommons.org/publicdomain/zero/1.0/ */
```
Comment (saschanaz) [inline scripts/migrations/002-remove-webview-flags.test.js:3]: Are the migrations/ things still needed? @queengooborg

Comment (queengooborg) [reply scripts/migrations/002-remove-webview-flags.test.js:3]: We aren't really using the migration scripts for anything, but I'll defer to @ddbeck on whether we should remove them.

File: scripts/release-stats.js [hunk 2]
```diff
@@ -30,25 +27,21 @@ const getJSON = url =>
...
 };
 
-const confirm = str => !['n', 'no'].includes(str.toLowerCase());
-
 const prompt = async questions => {
   const results = {};
   for (const q of questions) {
-    const options = q.type === confirm ? '(Y/n) ' : '';
-    results[q.name] = await question(`${q.message} ${options}`).then(q.type);
+    results[q.name] = await question(q.message).then(q.type);
```
Comment (ddbeck) [inline scripts/release-stats.js:44]: Can I have the trailing space back? Looks a little nicer 😄

```suggestion
    results[q.name] = await question(`${q.message} `).then(q.type);
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
A couple little things

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**